File: Blog-web-service/src/main/java/com/blog/dao/UserMapper.java
package com.blog.dao;

import com.blog.pojo.User;
import tk.mybatis.mapper.common.Mapper;

/**
 * @author JeungNyeongJae
 */

@org.apache.ibatis.annotations.Mapper
public interface UserMapper extends Mapper<User> {
}

File: Blog-web-service/src/main/java/com/blog/dao/UserMapper.kt
package com.blog.dao

import com.blog.pojo.User
import tk.mybatis.mapper.common.Mapper

/**
 *  @author JeungNyeongJae
 *  @date 2019/11/7
 *
 */
@org.apache.ibatis.annotations.Mapper
interface UserMapper : Mapper<User>
